packages/shared/src/utils/runtimeMode.ts

```diff
@@ -0,0 +1,14 @@
+export const isDevelopmentMode = (): boolean => {
```

Comment:
Nit: Node.js calls this "development and production environments" (https://nodejs.dev/en/learn/nodejs-the-difference-between-development-and-production/). So a suggestion for another name would be `isDevelopmentEnvironment`

Reply:
helper renamed. ptal